clients/cobol-lsp-vscode-extension/src/commands/RenumCommand.ts

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2025 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+import * as vscode from "vscode";
+
+export enum RenumberAction {
+  LEFT,
+  RIGHT,
+}
+
+const maxLines = 999999;
+
+const leftStart = 0;
+const leftEnd = 6;
+const leftDigits = 6;
+const leftMultiplier = 100;
+
+const rightStart = 72;
+const rightEnd = 80;
+const rightMultiplier = 1000;
+const rightDigits = 8;
+/**
+ * Renumber 1-7 or 73-80 columns in active editor.
+ *
+ * @param actionType RenumberAction
+ */
+export async function renumberLines(
+  actionType: RenumberAction,
+  activeEditor: vscode.TextEditor | undefined = vscode.window.activeTextEditor,
+) {
+  if (!activeEditor) {
+    return;
+  }
+  await notificationHandler(
+    renum,
+    activeEditor,
+    actionType,
+    "Renumbering Sequential Lines...",
+  );
+}
+/**
+ * Remove sequential numbers at 1-7 or 73-80 columns in active editor.
+ *
+ * @param actionType RenumberAction
+ */
+export async function unNumberLines(
+  actionType: RenumberAction,
+  activeEditor: vscode.TextEditor | undefined = vscode.window.activeTextEditor,
+) {
+  if (!activeEditor) {
+    return;
+  }
+
+  await notificationHandler(
+    unNumber,
+    activeEditor,
+    actionType,
+    "Removing Sequential Lines...",
+  );
+}
+
+async function unNumber(
+  textEditor: vscode.TextEditor,
+  actionType: RenumberAction,
+) {
+  const { start, end } =
+    actionType == RenumberAction.LEFT
+      ? {
```

Comment:
Extract these into a const objects. They could perhaps be even exported and used directly as arguments instead of RenumberAction.